team-config.yml

```diff
@@ -141,4 +141,12 @@ makeaplea:
     contact_channel: "#pet"
     build_notices_channel: "#pet-devops"
   tags:
-    application: makeaplea
\ No newline at end of file
+    application: makeaplea
+darts:
+  team: "Darts Modernisation"
+  namespace: "darts"
+  slack:
+    contact_channel: "#darts-devs"
+    build_notices_channel: "#darts-builds"
+  tags:
+    application: DARTS
```

Comment:
can you confirm this tag with `Bruce Rawstorne` on slack please?

I suspect at a minimum it would be lower case

Reply:
It's an acronym and other similarly namedprojects within this file are in uppercase, FYI

Reply:
the only other one is IAC-Fees, often these acronym's get expanded, this is what goes into finance reports so just need to make sure Bruce is happy with the tags.

If you can ping him and ask him about it i can approve for now and then you can update it?

Reply:
Have sent him a message, but currently showing as offline on slack.

Reply:
no response, so discussed with @jackmaloney and changed it to lower case

Reply:
Hi
Would be good to talk about what is being built for "darts" as might be one of those that we keep uppercase as everyone know it as that.  Let's chat on Slack.